### DISCUSSION TITLE
Enabling auto complete for theme object

### DISCUSSION THREAD

**Discussion by drgx** [ben-rogerson/twin.macro#474] [Q&A]
Hi Team,
I find that I cannot do autocomplete via IntelliSense for the "theme" object. but it works for  "tw" class names. 
Any suggestion for solving this issue? 
I can manually load the theme object directly via TailwindConfig but it kinda defeats the purpose of the `theme` object on the twin.macro.

https://user-images.githubusercontent.com/5230095/125044561-46befc00-e0c6-11eb-990c-c993066e090b.mp4

Thank you!

**Comment by ben-rogerson** [comment]
Heya Ryan,

Twin doesn't provide an IntelliSense plugin - you're likely using one of these two vscode plugins:

- [tailwindlabs/tailwindcss-intellisense](https://github.com/tailwindlabs/tailwindcss-intellisense)
- [lightyen/vscode-tailwindcss-twin](https://github.com/lightyen/vscode-tailwindcss-twin)

Cheers
